src/content/developers/docs/smart-contracts/verifying/index.md

```diff
@@ -104,5 +104,5 @@ Using the Tenderly Hardhat plugin allows for more control over the verification
 
 ## Further reading {#further-reading}
 
-- [How to verify the source code of Ethereum smart contract](https://developpaper.com/how-to-verify-the-source-code-of-ethereum-smart-contract/)
+- [How to verify the source code of Ethereum smart contract](https://ethereum.org/en/developers/docs/smart-contracts/verifying/)
```

Comment:
Use relative links when linking internally: https://ethereum.org/en/contributing/style-guide/content-standardization/#internal-links

```suggestion
- [How to verify the source code of Ethereum smart contract](/developers/docs/smart-contracts/verifying/)
```

Reply:
Actually, this link references the page it is on.... if this link is dead, we should just remove it 👍

Reply:
> Actually, this link references the page it is on.... if this link is dead, we should just remove it 👍

understood 😁